gradle.properties

```diff
@@ -0,0 +1,16 @@
+GROUP=com.instacart.formula
+VERSION_NAME=0.5.2
+
+POM_DESCRIPTION=Formula
+
+POM_URL=https://github.com/instacart/formula/
+POM_SCM_URL=https://github.com/instacart/formula/
+POM_SCM_CONNECTION=scm:git:git://github.com/instacart/formula.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/instacart/formula.git
+
+POM_LICENCE_NAME=BSD 3-Clause Clear License
+POM_LICENCE_URL=https://raw.githubusercontent.com/instacart/formula/master/LICENSE
+POM_LICENCE_DIST=repo
+
+POM_DEVELOPER_ID=instacart
+POM_DEVELOPER_NAME=Instacart
```

Comment:
Does the distribution URL default to mavenCentral? I would assume so.

Reply:
The plugin we use defaults to that. https://github.com/vanniktech/gradle-maven-publish-plugin